src/components/o-modal/index.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from '../../js/prop-types';
+import BaseComponentGlobal from '../../js/abstract/base-component-global';
+import defineOnce from '../../js/define-once';
+import styles from './index.scss';
+import template from './_template';
+
+class AXAModal extends BaseComponentGlobal {
+  static tagName = 'axa-modal';
+  static propTypes = {
+    open: PropTypes.bool,
+  };
+
+  set open(value) {
+    this.setAttribute('open', value);
+  }
+
+  get open() {
+    return this.getAttribute('open');
+  }
+
+  closeModal() {
+    this.removeAttribute('open');
+  }
+
+  closeModalByClickingOutside(event) {
+    if (event.target === this.querySelector('.js-modal')) this.closeModal();
+  }
+
+  closeModalOnEsc(event) {
+    const key = event.key || event.keyCode;
+    if (key === 'Escape' || key === 'Esc' || key === 27) {
+      this.closeModal();
+    }
+  }
+
+  init() {
+    super.init({ styles, template });
+  }
+
+  static get observedAttributes() {
```

Comment:
@raphaellueckl this method is always on top as every static property so that when u look at the file is immediatly visible what properties are accessible from outside. Please move it to the top

Reply:
Right, interesting that eslint does not complain about that.

Reply:
yepp